### ISSUE TITLE
Loss and Perplexity during train & eval

### ISSUE THREAD

**Issue by oliversms** [google-research/pegasus#29]
Will you add support for computing validation set perplexity? I dont see perplexity in the list of metrics. Also during training, its a bit hard to gage where we are in terms of train/eval loss/ppl so we can decide when to stop training. Just wondering if there is a plan to add this to the code?

**Comment by JingqingZ** [comment]
Hi you can collect perplexity (and other metrics) of validation/training set by running https://github.com/google-research/pegasus/blob/master/pegasus/bin/continuous_eval.py. This file is supposed to run regularly during training so that you can evaluate every couple of training steps. Alternatively, running https://github.com/google-research/pegasus/blob/master/pegasus/bin/evaluate.py with `--evaluate_test False` is also workable to evaluate validation set.

**Comment by oliversms** [comment]
Thankyou & closing!

**Comment by nliu86** [comment]
How do you run continuous_eval.py? Do we have to specify which GPU to use? I got OOM error.
